package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "oncilla",
-  "version": "2.2.0",
+  "version": "2.3.0",
```

I would prefer not to release a new version until the functionality is done. I don’t see auth message sending functionality in the oncilla-client part in this PR. Do we plan to do it separately, or should we include it in here?

More generally, I’d rather I bump the version number when I release it, to not get confused, so if we remove the change to the version number, we can merge this as is.

```suggestion
  "version": "2.2.0",
```